docs/source/release/v5.1.0/indirect_geometry.rst

```diff
@@ -22,43 +27,66 @@ Improvements
+- Added docking and undocking to the plot window and function browser window for the fit
+  tabs in Indirect Data Analysis on workbench.
+- Update the Indirect, Corrections user interface to use the
+  :ref:`PaalmanPingsMonteCarloAbsorption <algm-PaalmanPingsMonteCarloAbsorption>` algorithm and make it
+  work for elastic datasets.
+- OutputCompositeMembers and ConvolveOutputs have been added as options in the ConvFit tab of Indirect
+  Data Analysis.
 - Improved the responsiveness of the function browsers within the Indirect Data Analysis interface.
-- Added a lower bound (0.0 < height) to the Delta function defined in the ConvFit tab of the Indirect Data Analysis interface.
-- Raw data will now be plotted from the input workspace, rather than the fit workspace, allowing the full range of data to be seen irrespective of the fitting bounds.
+- Added a lower bound (0.0 < height) to the Delta function defined in the ConvFit tab of the Indirect
+  Data Analysis interface.
+- In Indirect Data Analysis interface, raw data will now be plotted from the input workspace, rather
+  than the fit workspace, allowing the full range of data to be seen irrespective of the fitting bounds.
 - Added fit parameter estimations to the MSD fitting tab within the Indirect Data Analysis interface.
-- Added the fit output information (Algorithm status and Chi squared) to each fitting tab of the Indirect Data Analysis interface.
-  This change introduces two optional outputs from the QENSFit algorithms (fit status and Chi squared), which may be used to monitor the outcome of the fit.
-- Added default parameter estimations to the F(q) tab.
-- The ConvFit tab within the IDA GUI will now output convolved members by default.
-- Added a Help option to the right-click menu in the function browser (in full function view) which brings up a relevant documentation page describing the function.
+- Added the fit output information (Algorithm status and Chi squared) to each fitting tab of the
+  Indirect Data Analysis interface.
+  This change introduces two optional outputs from the QENSFit algorithms (fit status and Chi squared),
+  which may be used to monitor the outcome of the fit.
+- Default parameter estimations have been added to the F(q) tab of the Indirect Data Analysis interface.
+- The ConvFit tab within the Indirect Data Analysis interface will now output convolved members by default.
+- Added a Help option to the right-click menu in the function browser (in full function view) which
+  brings up a relevant documentation page describing the function.
 
 Bug Fixes
 #########
 
-- :ref:`CalculateMonteCarloAbsorption <algm-CalculateMonteCarloAbsorption>` will now work correctly for annular sample in a container.
-- FQ and Msd tabs now label output workspaces with the fitting function.
+.. figure:: ../../images/Indirect_Data_Analysis_5_1_0.png
+   :class: screenshot
+   :width: 700px
+   :align: right
```

Comment:
Is this just the same image repeated??

Reply:
misstype  one should be a different name.